app/generator/print-styles.js

```diff
@@ -0,0 +1,93 @@
+const { millimetresToPoints } = require('./conversion')
+
+const zeroMillimeters = 0
+const fiveMillimeters = 5
+const tenMillimeters = 10
+const twentyFiveMillimeters = 25
+const thirtyFiveMillimeters = 35
+const twoHundredMillimeters = 200
```

Comment:
again I'm going to be annoying, but could these constants be named to reflect their purpose? e.g. const topMargin = xx